.travis.yml

```diff
@@ -43,7 +43,13 @@ matrix:
       # Strict compilation requires more than 2 GB
       script: MAVEN_OPTS='-Xmx3000m' mvn clean -Pstrict -pl '!benchmarks' compile test-compile spotbugs:check -B --fail-at-end
 
-    # packaging check
+      # analyze dependencies
```

Comment:
Please add a comment here w/ rationale as to why we are doing this. (People will appreciate being able to read that if the job fails on one of their PRs.)

Something that mentions what can go wrong if you use undeclared transitive deps, and what can go _right_ if you avoid declaring deps you're not using.

Reply:
That's a great suggestion. I'm thinking of using the `after_failure` travis lifecycle step to print a helpful message at the end of the log.